test/e2e/framework/test_context.go

```diff
@@ -484,7 +484,7 @@ func AfterReadingAllFlags(t *TestContextType) {
 	var err error
 	TestContext.CloudConfig.Provider, err = SetupProviderConfig(TestContext.Provider)
 	if err != nil {
-		if os.IsNotExist(errors.Cause(err)) {
+		if os.IsNotExist(errors.Unwrap(err)) {
```

Comment:
I wonder whether this should be `errors.Is(err, os.ErrNotExist)` as in https://golang.org/pkg/errors/#example_Is

But `os.IsNotExist` also returns true for some other errors, so it's not quite the same.

Let's keep it as-is.